.gitignore

```diff
@@ -1 +1,2 @@
 config/*.json
+config/.env*
```

Comment:
Would you be able to:

1.  Commit a `./config/.env.example` with the env vars we'll want?
1.  Add an example `docker build --no-cache` command to the README?

Reply:
1. Will do.
2. Will do. 😄